site/hi/index.html

```diff
@@ -0,0 +1,669 @@
+                    <p class="regular">एक्लिप्स कलेक्शन 8.0.0 <a href="https://github.com/eclipse/eclipse-collections/releases/tag/8.0.0">प्रकाशित</a> <a class="light">(8 सितंबर 2016)</a></p>
+                </div>
+                <div class="col-lg-4 col-sm-4">
+                    <h3>सामुदायिक लिंक</h3>
+                        <p class="regular">एक्लिप्स पीएमआई: <a href="https://projects.eclipse.org/projects/technology.collections">एक्लिप्स परियोजना प्रबंधन आधारिक संरचना</a></p>
+                        <p class="regular">गिटहब: <a href="https://github.com/eclipse/eclipse-collections">एक्लिप्स-कलेक्शन</a></p>
+                        <p class="regular">मुद्दे: <a href="https://github.com/eclipse/eclipse-collections/issues">गिटहब मुद्दे</a></p>
+                        <p class="regular">ईमेल की सूची: <a href="https://dev.eclipse.org/mailman/listinfo/collections-dev">कलेक्शन-देव</a> मेलिंग सूची</p>
+                        <p class="regular">विकी: <a href="https://github.com/eclipse/eclipse-collections/wiki">गिटहब विकि</a></p>
+                        <p class="regular">मंच: <a href="https://www.eclipse.org/forums/index.php?t=thread&amp;frm_id=329">एक्लिप्स कलेक्शन मंच</a></p>
+                        <p class="regular">स्टाकॉवेर्फ्लो: टैग के साथ <a href="http://stackoverflow.com/questions/tagged/eclipse-collections">एक्लिप्स-कलेक्शन</a></p>
+                </div>
+                <!-- /.col -->
+            </div>
+            <!-- /.row -->
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- Container Types -->
+
+
+    <div class="banner">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-12">
+                    <h3>- अधिक सुविधाएँ, कम अपशिष्ट -</h3>
+                </div>
+            </div>
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- /.banner -->
+
+    <!-- Footer -->
+    <footer>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-12">
+                    <ul class="list-inline">
+                        <li>
+                            <p class="copyright text-muted small">कॉपीराइट &copy; 2018 एक्लिप्स फाउंडेशन. सर्वाधिकार सुरक्षित. </p>
+                        </li>
+                        <li class="footer-menu-divider">&sdot;</li>
+                        <li>
+                            <a class="small" href="http://www.eclipse.org">एक्लिप्स फाउंडेशन</a>
+                        </li>
+                        <li class="footer-menu-divider">&sdot;</li>
+                        <li>
+                            <a class="small" href="http://www.eclipse.org/legal">कानूनी</a>
+                        </li>
+                        <li class="footer-menu-divider">&sdot;</li>
+                        <li>
+                            <a class="small" href="http://www.eclipse.org/legal/privacy.php">गोपनीयता</a>
```

Comment:
this means confidential. I dont quite know what it is for privacy.

Reply:
literally type "privacy" in Hindi here. While gopneeyatha means privacy, the translation does not work.

Reply:
done